### ISSUE TITLE
[user_status] Status write error message

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/server#24151]
### Steps to reproduce

1. Open the Dashboard.
2. Click on the status change button.
3. Select the status, for example: "Invisible".
4. Click "Set status message".

### Expected behaviour
Correct setting of the status by clicking on the "Set status message" button. Currently, the change works only after closing the window with status settings.

![obraz](https://user-images.githubusercontent.com/47037905/99241548-4ce22e80-27fe-11eb-8dd2-1951888554e3.png)

### Actual behaviour
The message "There was an error saving the status" is displayed.

![obraz](https://user-images.githubusercontent.com/47037905/99241364-10aece00-27fe-11eb-88ea-0447473a6c04.png)

### Server configuration

**Nextcloud version:** (see Nextcloud admin page)
20.0.1

**Dashboard version:**
7.0.0

**Comment by tobiasKaminsky** [comment]
> My opinion after clicking on the status, e.g.: "Invisible", the window for setting the status should close automatically.

The idea is that if you want to have "dnd" and a custom message, e.g. "in an important meeting" you can do this in one step.
With your change it would require two steps.

**Comment by Valdnet** [comment]
> With your change it would require two steps.

@tobiasKaminsky You're right. But in this case it would be possible to save the set status without any additional description using the "Set status message" button. As it is done in the Nextcloud Android app.

![obraz](https://user-images.githubusercontent.com/47037905/99414664-54d1c980-28f7-11eb-89ff-8d5561cb1d07.png)

In the current version for Nextcloud, this is confusing for users. When trying to save the set status, e.g.: "Invisible" without any additional description set, the message "There was an error saving the status" is displayed.

![obraz](https://user-images.githubusercontent.com/47037905/99414696-5f8c5e80-28f7-11eb-8bd4-2e048e542a56.png)

**Comment by tobiasKaminsky** [comment]
> hen trying to save the set status, e.g.: "Invisible" without any additional description set, the message "There was an error saving the status" is displayed.

@nextcloud/designers 
This then is misleading or a bug…
On our cloud I can change to any status and just click on the background to close the status popup and it works.
Same when choosing "invisible" and clicking on "set status".

**Comment by skjnldsv** [comment]
Changind online status and status message doesn't have the same ux.
Just click the online status saves it.
Clicking the Status message button, well, saves the status message.
But it should indeed be improved and the button should be disabled when the input is empty.

Adding a saved feedback on the online status buttons should be a nice addition too!
